matchzoo/models/arci_model.py

```diff
@@ -0,0 +1,80 @@
+"""An implementation of ArcI Model."""
+from matchzoo import engine
+import numpy as np
+from keras.models import Model
+from keras.layers import Input, Embedding, Conv1D, MaxPooling1D, Dropout, \
+                         Concatenate, Flatten
+
+
+class ArcIModel(engine.BaseModel):
+    """
+    ArcI Model.
+
+    Examples:
+        >>> model = ArcIModel()
+        >>> model.guess_and_fill_missing_params()
+        >>> model.build()
+
+    """
+
+    @classmethod
+    def get_default_params(cls) -> engine.ParamTable:
+        """:return: model default parameters."""
+        params = super().get_default_params()
+        params['optimizer'] = 'adam'
+        params['input_shapes'] = [(32,), (32,)]
+        params['input_dtypes'] = [np.int32, np.int32]
+        params.add(engine.Param('maxlen_left', 32))
+        params.add(engine.Param('maxlen_right', 32))
+        params.add(engine.Param('embed_finetune', False))
+        params.add(engine.Param('embed_dim', 300))
+        params.add(engine.Param('vocab_size', 100))
+        params.add(engine.Param('left_kernel_count', 32))
+        params.add(engine.Param('left_kernel_size', 3))
+        params.add(engine.Param('right_kernel_count', 32))
+        params.add(engine.Param('right_kernel_size', 3))
+        params.add(engine.Param('q_pool_size', 16))
+        params.add(engine.Param('d_pool_size', 16))
```

Comment:
`q_pool_size` denotes for  `query_pool_size`? then why not `left_pool_size`? same for `d_pool_size`